webapp/pubspec.yaml

```diff
@@ -5,11 +5,17 @@ environment:
   sdk: ^2.7.0
 
 dependencies:
-  firebase: ^7.2.0
+  firebase: ^5.0.0
   http: ^0.12.0
   intl: ^0.16.0
   chartjs:
     git: https://github.com/marianamarasoiu/chartjs.dart.git
+  nook:
+    git:
+      url: https://github.com/larksystems/nook.git
```

Comment:
I always use a `git` url as in `git@github.com:larksystems/nook.git`
but that's out of habbit... I don't know which one is better or more common.